assets/js/custom/Follower.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-env jquery */
+/* global Swal */
+
+// eslint-disable-next-line no-unused-vars
+function Follower (unfollowUrl, followUrl, somethingWentWrongError, followError, unfollowError,
+  visibleFollowing = 5, visibleFollowers = 5, showSte = 5,
+  minAmountOfVisibleFollowers = 5, totalFollowing = 5, totalFollowers = 5) {
+  const self = this
+  self.unfollowUrl = unfollowUrl
+  self.followUrl = followUrl
+  self.somethingWentWrongError = somethingWentWrongError
+  self.followError = followError
+  self.unfollowError = unfollowError
+
+  self.unfollow = function (id, username) {
+    const $followerItem = $('.follower-item-' + id)
+    const $buttons = $followerItem.find('.follow-button button').attr('disabled', true)
```

Comment:
Could be misleading. We're talking about 1-2 `.follower-item`s, not just one.
```suggestion
    const $followerItems = $('.follower-item-' + id)
    const $buttons = $followerItems.find('.follow-button button').attr('disabled', true)
```